lib/openssl/asn1.rb

```diff
@@ -10,6 +10,41 @@
 
 module OpenSSL
   module ASN1
+    V_ASN1_UNIVERSAL= 0x00
+    V_ASN1_APPLICATION = 0x40
+    V_ASN1_CONTEXT_SPECIFIC = 0x80
+    V_ASN1_PRIVATE = 0xc0
+    V_ASN1_CONSTRUCTED = 0x20
+    V_ASN1_PRIMITIVE_TAG = 0x1f
```

Comment:
IMO these constants are not useful and it's better to simply embed these values.

Reply:
removed.